src/styling/HomePage.css

```diff
@@ -3,27 +3,52 @@
   src: url(Cornerstone.ttf);
 }
 
-.first-row {
-  margin-top: 150px;
+.logotype {
+  font-family: cornerstone;
+  color: #f8ed44;
```

Comment:
can we use the css vars for this? there's a couple other places in the rest of this file too

Reply:
fixed!